README.md

```diff
@@ -219,6 +219,7 @@ Authors
 - Dr. Antonio Di Pilato
 - Simeon Ehrig
 - Bernhard Manfred Gruber*
+- Dr. Axel Huebl
```

Comment:
It renders all fine, so not need to fix. But part of this bullet list is misaligned relative to its start.